person.rb

```diff
@@ -0,0 +1,32 @@
+require './nameable'
+require './captalize_decorator'
+require './trimmer_decorator'
```

Comment:
- [ ] Please know that these 2 lines are causing issues because the files don't exist, the files you want to require are named differently in your project, kindly fix this issue.

Reply:
Alright!